<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>593</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Auswerte-Pipeline</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0">
     <item row="18" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_start_analysis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Start Analysis</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="label_model_path">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>9</height>
        </size>
       </property>
       <property name="text">
        <string>None</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_model">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The software allows to specify pretrained classification models generated based on the Model_class.py file in the code folder.&lt;/p&gt;&lt;p&gt;This is intended for advanced users.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Classification model</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="cbox_cell_classification_f1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_cell_classification_f1_ch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ch:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbox_cell_classification_f1_ch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_threads">
       <property name="toolTip">
        <string>If set to &gt;1 multiprocessing is used. High values may lead to 'out of memor' errors during calculation.</string>
       </property>
       <property name="text">
        <string>No. of threads</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spin_nuc_ch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Channel number of the nuclei signal. Starting from zero.</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="edit_ch_thresh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Threshold values for each channel given as '[t_c1, t_c2, ...]'. This is only used for the calculation of signal_sum_foreground.</string>
       </property>
       <property name="text">
        <string>[25, 25]</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_seg_identifier">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CellACDC: file name identifier after '..._segm_IDENTIFIER'.&lt;/p&gt;&lt;p&gt;Raw_Data: file name identifier before image name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Segmentation identifier</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QVBoxLayout" name="vert_layout_add_seg_ident">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btn_add_seg_ident_minus">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add_seg_ident_plus">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edit_max_vol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Nuclei with volumes above this value are discarded. Set to -1 to not use this feature.</string>
       </property>
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="btn_model_path">
         <property name="toolTip">
          <string>Pretrained models can be loaded to classifiy nuclei.</string>
         </property>
         <property name="text">
          <string>Select Model</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_unselect_model">
         <property name="text">
          <string>Remove Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_cell_classification_comperator">
         <property name="text">
          <string>&gt;=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_cell_classification_threshold"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="edit_seg_identifier">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CellACDC: file name identifier after '..._segm_IDENTIFIER'.&lt;/p&gt;&lt;p&gt;Raw_Data: file name identifier before image name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_cell_classification_norm">
       <property name="text">
        <string>Normalization (f1/f2)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_cell_classification">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The software allows for a basic object classification based on feature thresholding. The objects are assigned to either the positive or negative class depending on the value of the selected feature (f1).&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Additionally a second feature (f2) can be used to normalize the thresholded feature: f_norm = f1/f2.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Cell classification</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_cell_classification_f1">
         <property name="text">
          <string>f1:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_cell_classification_f2">
         <property name="text">
          <string>f2:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_ch_thresh">
       <property name="toolTip">
        <string>Threshold values for each channel given as '[t_c1, t_c2, ...]'. This is only used for the calculation of signal_sum_foreground.</string>
       </property>
       <property name="text">
        <string>Channel thresholds</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_add_seg_ident">
       <property name="toolTip">
        <string>The software allows to utilize other segmentations to analyze basic object features. Use the + - buttons to add or remove input fields.</string>
       </property>
       <property name="text">
        <string>Additional segmentation identifier</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QComboBox" name="cbox_cell_classification_f2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_cell_classification_f2_ch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ch:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbox_cell_classification_f2_ch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spin_threads">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>If set to &gt;1 multiprocessing is used. High values may lead to 'out of memor' errors during calculation.</string>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_max_vol">
       <property name="toolTip">
        <string>Nuclei with volumes above this value are discarded. Set to -1 to not use this feature.</string>
       </property>
       <property name="text">
        <string>Minium nuclei volume [µm^3]</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_nuc_ch">
       <property name="toolTip">
        <string>Channel number of the nuclei signal. Starting from zero.</string>
       </property>
       <property name="text">
        <string>Nuclei channel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_min_vol">
       <property name="toolTip">
        <string>Nuclei with volumes below this value are discarded. Set to -1 to not use this feature.</string>
       </property>
       <property name="text">
        <string>Minium nuclei volume [µm^3]</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edit_min_vol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Nuclei with volumes below this value are discarded. Set to -1 to not use this feature.</string>
       </property>
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbox_data_structure">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The GUI supports two folder structures. The first Cell_ACDC is based on the folder structure and file names of Cell-ACDC.&lt;br/&gt;(https://github.com/SchmollerLab/Cell_ACDC).&lt;/p&gt;&lt;p&gt;The second Raw_Data expects the following folder structure:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Menlo','Monaco','Courier New','monospace'; font-size:12px; color:#cccccc;&quot;&gt;├── Dataset&lt;br/&gt;│   ├── Raw_Data&lt;br/&gt;│   │   ├── ConditionA&lt;br/&gt;│   │   │   ...&lt;br/&gt;│   │   ├── ConditionB&lt;br/&gt;│   │   │   ...&lt;br/&gt;│   ├── Segmentation&lt;br/&gt;│   │   ├── ConditionA&lt;br/&gt;│   │   │   ...&lt;br/&gt;│   │   ├── ConditionB&lt;br/&gt;│   │   │   ...&lt;br/&gt;│   ...&lt;br/&gt;...&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The folder structure inside Raw_Data and Segmentation can be choosen arbitrarily, e.g., multiple subfolders, as long as they are consistent between the Raw_Data and Segmentation folder.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_input_folder">
       <property name="toolTip">
        <string>Select the top folder containing all the folders and image files to be processed.</string>
       </property>
       <property name="text">
        <string>Select Input Folder</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_data_structure">
       <property name="text">
        <string>Data structure</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_input_folder">
       <property name="text">
        <string>None</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spin_threads</tabstop>
  <tabstop>edit_min_vol</tabstop>
  <tabstop>edit_max_vol</tabstop>
  <tabstop>spin_nuc_ch</tabstop>
  <tabstop>edit_ch_thresh</tabstop>
  <tabstop>edit_seg_identifier</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
